AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -194,8 +201,10 @@ + (void)initialize
     // Subclasses should never override these
```

Comment:
Optional: We should probably only calculate `NSStringFromClass(self)` once in this method and then supply that to the `ASDisplayNodeAssert()` calls